Source/Core/Core/HW/WiimoteEmu/Extension/Classic.cpp

```diff
@@ -177,26 +173,27 @@ void Classic::Reset()
   m_reg.identifier = classic_id;
 
   // Build calibration data:
+  // All values are to 8 bits of precision.
   m_reg.calibration = {{
       // Left Stick X max,min,center:
-      CAL_STICK_CENTER + CAL_STICK_RANGE,
-      CAL_STICK_CENTER - CAL_STICK_RANGE,
+      CAL_STICK_CENTER + STICK_GATE_RADIUS,
+      CAL_STICK_CENTER - STICK_GATE_RADIUS,
       CAL_STICK_CENTER,
       // Left Stick Y max,min,center:
       CAL_STICK_CENTER + CAL_STICK_RANGE,
```

Comment:
I believe this should be:
```suggestion
       CAL_STICK_CENTER + STICK_GATE_RADIUS,
```

Reply:
Whoops. Nice catch. Thanks.